mlflow/deployments/cli.py

```diff
@@ -220,3 +237,24 @@ def run_local(flavor, model_uri, target, name, config):
     """
     config_dict = _user_args_to_dict(config)
     interface.run_local(target, name, model_uri, flavor, config_dict)
+
+
+@commands.command("predict")
+@parse_custom_arguments
+@deployment_name
+@target_details
+@parse_input
+@parse_output
+def predict(target, name, input_path, output_path, config):
+    """
+    Predict the results for the deployed model for the given input(s)
+    """
+    config_dict = _user_args_to_dict(config)
+    df = pd.read_json(input_path)
+    client = interface.get_deploy_client(target)
+    result = client.predict(name, df, config_dict)
+    click.echo("\n")
```

Comment:
A few comments:
* Could we check here whether an output path is specified, and if so not `click.echo` the output?
* Also, when we print the output, could we just print the raw output, rather than wrapping it in newlines and a "RESULT IS" header? 
* We should make sure to print the output in JSON format

See the implementation of `mlflow models predict` [here](https://github.com/mlflow/mlflow/blob/00c7d0bdbb1d1f082abdb4107d2960ab26681381/mlflow/pyfunc/scoring_server/__init__.py#L234-L250) for reference

Reply:
Done. Addressed the above comments.